omero/sysadmins/unix/install-web/web-deployment.rst

```diff
@@ -57,12 +53,6 @@ more specific walkthrough listed below.
         Instructions for installing **separately** OMERO.web from
         scratch on Debian 9 with Ice 3.6.
 
-    :doc:`walkthrough/omeroweb-install-osx-ice3.6`
-        Instructions for installing OMERO.web from scratch on
-        on Mac OS X with Homebrew. It is aimed at **developers**
-        since typically MacOS X is not suited for serious deployment.
-
-
```

Comment:
Not related to this PR, but I wonder if we could remove the ```.. seealso::``` in these sections (on the main server and web install pages). They make the links that follow seem optional, but really they are the essential guides.